Character.h

```diff
@@ -53,16 +54,17 @@ struct Character
         std::cout << getName() << "'s attack damage level has been boosted to " << attackDamage << std::endl;
     }
 
-    void printStats()
-    {
+    void printStats();
+    /*{
         std::cout << getName() << "'s stats: " << std::endl;
         std::cout << getStats(); //make your getStats() use a function from the Utility.h
         
         std::cout << std::endl;
         std::cout << std::endl;
-    }
+    }*/
 protected:
     std::vector<std::unique_ptr<Item>> defensiveItems;
+    std::vector<std::unique_ptr<Item>> attackItems;
```

Comment:
The attack item is not a part of the Character class.
The `DragonSlayer` is the only one who gets ONE attack item:
```
7) create a new Item-derived type that boosts the attack damage called AttackItem 
        add an instance of it to the DragonSlayer class
        make the DragonSlayer::attack use it when the dragonSlayer attacks a dragon. 
 ```
- [ ] revise